Dockerfile

```diff
@@ -23,8 +23,8 @@ ARG BURN_BUILDER_GOLANG=@sha256:f7d3519759ba6988a2b73b5874b17c5958ac7d0aa48a8b1d
 ARG GPROFILER_BUILDER_UBUNTU=@sha256:cf31af331f38d1d7158470e095b132acd126a7180a54f263d386da88eb681d93
 # node-package-builder-musl alpine
 ARG NODE_PACKAGE_BUILDER_MUSL=@sha256:69704ef328d05a9f806b6b8502915e6a0a4faa4d72018dc42343f511490daf8a
-# node-package-builder-glibc - ubuntu:20.04
-ARG NODE_PACKAGE_BUILDER_GLIBC=@sha256:cf31af331f38d1d7158470e095b132acd126a7180a54f263d386da88eb681d93
+# node-package-builder-glibc - centos7 with gcc 7
```

Comment:
"centos7 with gcc 7" - let's replace it with the image:tag you got the hash from?

Reply:
(I suppose `centos/devtoolset-7-toolchain-centos7:latest`?)